### PR TITLE
Add entry in additions/removals about parameters in CREATE/ALTER DB TOPOLOGY

### PR DESCRIPTION
**Pull request by l-heemann**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1082.surge.sh

Updated pages:
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-1082.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

**Review by HannesSandberg — APPROVED** [review]
lgtm 👍 

note: there will be a merge conflict between this one and https://github.com/neo4j/docs-cypher/pull/1070

File: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc [hunk 1]
```diff
@@ -16,6 +16,29 @@ New features are added to the language continuously, and occasionally, some feat
 This section lists all of the features that have been removed, deprecated, added, or extended in different Cypher versions.
 Replacement syntax for deprecated and removed features are also indicated.
 
+[[cypher-deprecations-additions-removals-5.26]]
+== Neo4j 5.26
+
+[cols="2", options="header"]
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc]:
```suggestion
== Neo4j 5.26

=== Updated features
[cols="2", options="header"]
```

File: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc [hunk 2]
```diff
@@ -16,6 +16,29 @@ New features are added to the language continuously, and occasionally, some feat
...
+[source, cypher, role="noheader"]
+----
+CREATE DATABASE foo TOPOLOGY $p PRIMARIES $s SECONDARIES
+----
+[source, cypher, role="noheader"]
+----
+ALTER DATABASE foo SET TOPOLOGY $p PRIMARIES $s SECONDARIES
+
+----
+| The `CREATE DATABASE` and `ALTER DATABASE` commands now accept parameters for `TOPOLOGY` configuration.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc]:
```suggestion
| The link:{neo4j-docs-base-uri}/operations-manual/{page-version}/database-administration/standard-databases/create-databases/[`CREATE DATABASE`] and link:{neo4j-docs-base-uri}/operations-manual/{page-version}/database-administration/standard-databases/alter-databases/[`ALTER DATABASE`] commands now accept parameters for `TOPOLOGY` configuration.
```

**Review by JPryce-Aklundh — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**